layouts/shortcodes/home-section-resources.html

```diff
@@ -0,0 +1,65 @@
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Christopher Guindon <chris.guindon@eclipse-foundation.org>
+
+  SPDX-License-Identifier: EPL-2.0
+-->
+
+<section class="container europe-resources">
+  <h2>Resources</h2>
+  <div class="row">
+    <div class="col-md-6 col-sm-12">
+      <div class="text-center match-height-item background-white padding-20 margin-bottom-30">
+        <p><img class="img img-responsive margin-auto padding-30" style="height: 150px; width: 150px;" src="images/resources/whitepaper-icon.svg" alt="resource image"></p>
+        <h3>Whitepaper</h3>
+        <p>Learn how open source is fueling digital transformation in Europe and beyond.</p>
```

Comment:
You'll want to put flex-grow on these paragraph tags. Currently, the buttons aren't aligned, and it makes the section look messy